From 761f11c8c8f3559558492ba580a6350af84c7455 Mon Sep 17 00:00:00 2001
From: sarthurdev <965089+sarthurdev@users.noreply.github.com>
Date: Tue, 28 Jul 2026 11:32:50 +0200
Subject: [PATCH] Enable FIPS module

---
 debian/openssl.install | 1 +
 debian/rules           | 2 ++
 2 files changed, 3 insertions(+)

diff --git a/debian/openssl.install b/debian/openssl.install
index 7068caa7cd..32a35449f5 100644
--- a/debian/openssl.install
+++ b/debian/openssl.install
@@ -5,6 +5,7 @@ usr/lib/ssl/certs
 usr/lib/ssl/private
 usr/lib/ssl/misc/*
 usr/lib/ssl/openssl.cnf
+usr/lib/ssl/fipsmodule.cnf
 usr/share/man/man1/*
 usr/share/man/man5/*
 usr/share/man/man7/*
diff --git a/debian/rules b/debian/rules
index 5a8097099e..49bea48a8c 100755
--- a/debian/rules
+++ b/debian/rules
@@ -114,6 +114,8 @@ override_dh_auto_install-indep:
 
 override_dh_auto_install-arch:
 	$(MAKE) -C build_shared install DESTDIR=`pwd`/debian/tmp
+	# Install FIPS module
+	$(MAKE) -C build_shared install_fips DESTDIR=`pwd`/debian/tmp
 	# pic static libraries, nobody should need them
 	cp -pf build_shared/libcrypto.a debian/tmp/usr/lib/$(DEB_HOST_MULTIARCH)/libcrypto.a
 	cp -pf build_shared/libssl.a debian/tmp/usr/lib/$(DEB_HOST_MULTIARCH)/libssl.a
-- 
2.50.1 (Apple Git-155)

